base16ct/src/lib.rs

```diff
@@ -0,0 +1,228 @@
+#![no_std]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![forbid(unsafe_code)]
+#![warn(missing_docs, rust_2018_idioms)]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
```

Comment:
I think it's worth to replace the URLs with `https://raw.githubusercontent.com/RustCrypto/media/6ee8e381/logo.svg` in the case if will move the logo for some reason.